ibm_catalog.json

```diff
@@ -0,0 +1,115 @@
+                            "key": "account_shell_settings_status"
+                        },
+                        {
+                            "description": "Current state of API key creation restriction",
+                            "key": "account_iam_apikey_creation"
+                        },
+                        {
+                            "description": "Current allowed IP addresses enforcement state",
+                            "key": "account_allowed_ip_addresses_enforced"
+                        },
+                        {
+                            "description": "Current MFA setting",
+                            "key": "account_iam_mfa"
+                        },
+                        {
+                            "description": "Current state of ServiceID creation restriction",
+                            "key": "account_iam_serviceid_creation"
+                        }
+                    ],
+                    "usage": ":exclamation: **Important:** Make sure that you set the `API_DATA_IS_SENSITIVE` environment variable to `true` to hide\nsensitive information before you run Terraform operations. For more information, see the generic REST API\nprovider [documentation](https://github.com/Mastercard/terraform-provider-restapi#usage):\n\nexport API_DATA_IS_SENSITIVE=true`\n\n##############################################################################\n# Config providers\n##############################################################################\n\ndata \"ibm_iam_auth_token\" \"tokendata\" {}\n\nprovider \"restapi\" {\n  uri                  = \"https:\"\n  write_returns_object = true\n  debug                = false # set to true to show detailed logs, but use carefully as it might print API key values.\n  headers              = {\n    Authorization = data.ibm_iam_auth_token.tokendata.iam_access_token\n    Content-Type  = \"application/json\"\n    if-Match      = \"*\"\n  }\n}\n\nprovider \"ibm\" {\n  ibmcloud_api_key = var.ibmcloud_api_key # pragma: allowlist secret\n}\n\n##############################################################################\n# Configure IAM Account settings\n##############################################################################\n\n# Replace \"master\" with a GIT release version to lock into a specific release\nmodule \"iam-account-settings\" {\n  source                       = \"git::https://github.com/terraform-ibm-modules/terraform-ibm-iam-account-settings?ref=main\"\n  allowed_ip_addresses         = var.allowed_ip_addresses\n  enforce_allowed_ip_addresses = true\n}"
+                }
+            ],
+            "keywords": [
+                "IaC",
+                "infrastructure as code",
+                "terraform",
+                "solution",
+                "iam",
+                "account settings"
+            ],
+            "label": "Terraform IAM Account Settings Module",
```

Comment:
I think this is the label for the flavor (variation) in the catalog. So would the label be `Standard` or something like that?

Reply:
oh yes good catch - I wonder what we should call the flavor label here. I'm wondering if maybe it should be `FSCloud`?

Reply:
It is a label for the offering. There is no label set for the flavor yet, I think I can add that to the flavor. So are we going ahead with `fscloud` or `standard`? 
<img width="1409" alt="image" src="https://user-images.githubusercontent.com/108140212/233332423-9a59826a-fe1c-47a6-9149-414a1bbc45fb.png">